_posts/2024-07-22-kubeflow-1.9-release.md

```diff
@@ -0,0 +1,226 @@
+
+## Fine-Tune APIs for LLMs
+
+In the rapidly evolving ML/AI landscape, the ability to fine-tune pre-trained models represents a significant leap towards achieving custom solutions with less effort and time. Fine-tuning with custom datasets allows practitioners to adapt large language models (LLMs) to their specific needs.
+
+However, fine-tuning tasks often require extensive manual intervention, including the configuration of training environments and the distribution of data across nodes. The new [Fine-Tune API](https://www.kubeflow.org/docs/components/training/user-guides/fine-tuning/) aims to simplify this process, offering an easy-to-use Python interface that abstracts away the complexity involved in setting up and executing fine-tuning tasks on distributed systems.
+
+By providing this API, Training Operator not only simplifies the user experience for ML practitioners but also leverages its existing infrastructure for distributed training. You can take advantage of Kubernetes’ ability to dynamically schedule GPU, thus saving on compute resources and cost. Training Operator also gives you fault tolerant guarantees, to save your training procedures from cluster node failures.
+
+## Pipelines
+
+### Argo Workflows and Tekton Backends Consolidation
+
+The Pipelines Tekton backend has been [merged](https://github.com/kubeflow/pipelines/pull/10678) into the main Kubeflow Pipelines repository. You can now choose what workflow engine to use from the same Pipelines version. This proves the extensibility and flexibility of the KFP v2 architecture, which encourages other contributors to bring support for other workflow engines.
+
+Compared to Argo Workflows, Tekton provides better extensibility thanks to pipeline definitions and reusable components that facilitate easier customization and integration into various CI/CD systems, making it adaptable for diverse machine learning workflows. Tekton also optimizes resource usage and execution efficiency, enhancing scalability for large-scale machine learning workloads compared to Argo Workflows, resulting in faster execution and better resource management.
+
+In this [blog post](https://developer.ibm.com/blogs/awb-tekton-optimizations-for-kubeflow-pipelines-2-0/) you can find more details about the benefits of running KFP with Tekton instead of Argo Workflows. 
+
+### Argo Workflows Upgrade
+
+Kubeflow Pipelines’s Argo Workflows backend is [upgraded to 3.4.16](https://github.com/kubeflow/pipelines/issues/10469). This upgrade moves the supported version closer to the latest upstream version and brings lots of CVE resolutions. The previous version was no longer being patched by the Argo community, so lots of security issues had accumulated over time.
+
+### v1 Feature Parity
+
+We made a huge progress towards KFPv1 feature parity by adding more Kubernetes  resources to the Pipelines code with the new kfp-kubernetes 1.2.0 Python package.
```

Comment:
this section might come 1st in the KFP section i.e. line 42.    We might preface it with an intro i.e. Kubeflow Pipelines is the most popular component in Kubeflow and over the last year the team has developed and matured the V2 version of KFP.   2nd question - comment - should we now recommend users to build new pipelines on v1 and implement their plans to migrate V1 pipelines to V2 ?

Reply:
moved and updated section